bundles/pixi.js/src/index.js

```diff
@@ -45,7 +45,7 @@ loaders.Loader.registerPlugin(textBitmap.BitmapFontLoader);
 loaders.Loader.registerPlugin(spritesheet.SpritesheetLoader);
 
 app.Application.registerPlugin(ticker.TickerPlugin);
-app.Application.registerPlugin(loaders.LoaderPlugin);
+app.Application.registerPlugin(loaders.AppLoaderPlugin);
```

Comment:
This looks like a functional change, is this intended?

Reply:
We have to separate ILoaderPlugin and AppLoaderPlugin. We have `LoaderPlugin` that is actually application plugin.

Reply:
@ivanpopelyshev and I discussed this. I'm find with that change.